src/plugins/intel_cpu/src/nodes/def_conv.cpp

```diff
@@ -681,6 +682,60 @@ bool DeformableConvolution::isSupportedOperation(const std::shared_ptr<const ngr
     return true;
 }
 
+namespace {
+
+struct DefConvKey {
+    std::vector<std::shared_ptr<BlockedMemoryDesc>> descVector;
+    DeformableConvolution::DefConvAttr defConvAttr;
+    impl_desc_type implType;
+
+    size_t hash() const;
+    bool operator==(const DefConvKey& rhs) const;
+};
+
+size_t DefConvKey::hash() const {
+    using namespace dnnl::impl;
+    using namespace dnnl::impl::primitive_hashing;
+
+    size_t seed = 0;
+
+    for (const auto& ptr : descVector) {
+        if (ptr) {
+            seed = get_vector_hash(seed, ptr->getBlockDims());
+            seed = get_vector_hash(seed, ptr->getStrides());
+            seed = get_vector_hash(seed, ptr->getOrder());
```

Comment:
Also we need to consider `OffsetPaddingToData` and `OffsetPadding` as well.

Reply:
Added